src/creator.rs

```diff
@@ -291,6 +299,8 @@ mod tests {
         finish(killers, handles).await;
     }
 
+    // This test checks if 5 creators that start at the same time and 2 creators that starts after those 5 first create 125 blocks,
```

Comment:
s/blocks/units (it's correct _almost_ everywhere :D)

Maybe mention in this comment that this is a test for catching up, it's relatively obvious from the statement, but it doesn't hurt to be more explicit.